.templates/blynk_server/directoryfix.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Create config files for Blynk custom server
+# 877dev
+
+#current user
+u=$(whoami)
+
+#Check if the config directory already exists:
+if [ ! -d ./volumes/blynk_server/data/config ]; then
+	#Create the config directory
+    sudo mkdir -p ./volumes/blynk_server/data/config
+
+    #Create the properties files:
+	#cd ~/IOTstack/volumes/blynk_server/data/config
+    sudo touch ./volumes/blynk_server/data/config/server.properties
+    sudo touch ./volumes/blynk_server/data/config/mail.properties
+
+    #Give permissions:
+    sudo chown -R $u:$u ./volumes/blynk_server/data/config
+
+    #Populate the server.properties file:
```

Comment:
While you can use double quote for multiline each, it might be better to use this method instead, incase a double quote is needed somewhere in the file:
```
#Populate the server.properties file:
cat <<EOT > ./volumes/blynk_server/data/config/server.properties
    hardware.mqtt.port=8440
    http.port=8080
    force.port.80.for.csv=false
    force.port.80.for.redirect=true
    https.port=9443
    data.folder=./data
    logs.folder=./logs
    log.level=info
    user.devices.limit=10
    user.tags.limit=100
    user.dashboard.max.limit=100
    user.widget.max.size.limit=20
    user.message.quota.limit=100
    notifications.queue.limit=2000
    blocking.processor.thread.pool.limit=6
    notifications.frequency.user.quota.limit=5
    webhooks.frequency.user.quota.limit=1000
    webhooks.response.size.limit=96
    user.profile.max.size=128
    terminal.strings.pool.size=25
    map.strings.pool.size=25
    lcd.strings.pool.size=6
    table.rows.pool.size=100
    profile.save.worker.period=60000
    stats.print.worker.period=60000
    web.request.max.size=524288
    csv.export.data.points.max=43200
    hard.socket.idle.timeout=10
    enable.db=false
    enable.raw.db.data.store=false
    async.logger.ring.buffer.size=2048
    allow.reading.widget.without.active.app=false
    allow.store.ip=true
    initial.energy=1000000
    admin.rootPath=/admin
    restore.host=blynk-cloud.com
    product.name=Blynk
    admin.email=admin@blynk.cc
    admin.pass=admin
EOT
```